File: test/unit/code_parser/whole_tests/sources/wgw_2010.rb
with_wg do |wg|
  comment "Perform the HTML to PDF conversion"
  DoPrint "127.0.0.1", @vars[:wg][:port], :assert_equal => 0
  if @useUnique == 1
    br
    comment "Use NewUniqueID to retrieve the unique filename used by WebGrabber"
    NewUniqueID :var => variable("uniqueFilename", "string")
  end
  br
  comment "Cleans up all internal string variables used during conversion"
  comment "By default variables are deleted 3 minutes after the conversion"
  CleanUp "127.0.0.1", @vars[:wg][:port]
end

File: test/unit/code_parser/whole_tests/views/wgw_2010.php
<?php
// Copyright (c) 2000 ActivePDF, Inc.
// ActivePDF WebGrabber WBE 2010
// Example generated 01/01/2000 
?>

<?php
// Instantiate Object
$oWG = new COM("APWebGrabber.Object");

// Perform the HTML to PDF conversion
$intDoPrint = $oWG->DoPrint("127.0.0.1", 58585);
if ($intDoPrint != 0) {
  Error("DoPrint", $intDoPrint);
}

// Cleans up all internal string variables used during conversion
// By default variables are deleted 3 minutes after the conversion
$oWG->CleanUp("127.0.0.1", 58585);

// Release Object
$oWG = null;

// Process Complete
echo "Done!";

// Error Handling
function Error($method, $outputCode) {
  echo "Error in " . $method . ": " . $outputCode;
}
?>